diff --git a/Makefile.config.in b/Makefile.config.in
index 707cfe0..fef031c 100644
--- a/Makefile.config.in
+++ b/Makefile.config.in
@@ -9,7 +9,7 @@ CXXFLAGS = @CXXFLAGS@
 CXXLTO = @CXXLTO@
 EDITLINE_LIBS = @EDITLINE_LIBS@
 ENABLE_S3 = @ENABLE_S3@
-GTEST_LIBS = @GTEST_LIBS@
+GTEST_LIBS = @GTEST_LIBS@ -lrapidcheck
 HAVE_LIBCPUID = @HAVE_LIBCPUID@
 HAVE_SECCOMP = @HAVE_SECCOMP@
 HOST_OS = @host_os@
